app/views/sessions/updatesession.scala.html

```diff
@@ -115,6 +115,22 @@
                     <div class="col-md-1" ></div>
                 </div>
 
+                <div class="col-md-12 login-third update-second">
```

Comment:
Would it make more sense to use youtubeURL and slideShareURL instead of youtubeLink and slideShareLink?